app/views/patients/data_entry.html.erb

```diff
@@ -40,7 +40,7 @@
 
   <%= f.text_field :age, autocomplete: 'off' %>
   <%= f.select :race_ethnicity, options_for_select(race_ethnicity_options), label: 'Race / Ethnicity', autocomplete: 'off' %>
-  <%= f.select :clinic_name, options_for_select(clinic_options) %>
+  <%= f.select :clinic_id, options_for_select(clinic_options) %>
```

Comment:
The select field `clinic_name` is looking for `patient#clinic_name`, while `clinic_id` is looking for `patient#clinic_id` (which, it turns out, is the field that gets quietly generated in the json when you set up a `belongs_to`, and activerecord is smart enough to translate `patient.clinic` into `Clinic.find(patient.clinic_id)`. 

I wound up having to use `clinic_id` here and not `clinic` because the latter does a lookup (as in, runs `Clinic.find(id)`) and chokes on nil values.

This post brought to you by learning this the hard way this afternoon 😑